modules/promotions/promotion-data.php

```diff
@@ -52,6 +56,74 @@ private static function get_animated_headline_data( $assets_data ) {
 		return self::filter_data( Utils::ANIMATED_HEADLINE, $data );
 	}
 
+	private static function get_video_playlist_data( $assets_data ) {
+		$data = [
+			'image' => esc_url( $assets_data[ Utils::VIDEO_PLAYLIST ] ) ?? '',
```

Comment:
@elchugreeva we need to make a micro-fix for `esc_url( ... ) ?? ''` => `esc_url( ... ?? '' )`